pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -232,6 +245,27 @@ def _configure_fault_tolerance_callbacks(self):
         # don't use `log_dir` to minimize the chances of failure
         self.trainer.callbacks.append(_FaultToleranceCheckpoint(dirpath=self.trainer.default_root_dir))
 
+    def _configure_external_callbacks(self) -> None:
```

Comment:
does this become easier to test if you make this a function which returns the callbacks from the entry point?
like this?
```
def _get_external_callbacks(self) -> List[Callback]:
```

and then in L105, you can do this:
```
self.trainer.callbacks.extend(self._get_external_callbacks())
```

Reply:
I followed your suggestion to make a function instead of a method, but kept the tests the way they are because I still see value in testing the insertion and position of these callbacks in the global list of trainer callbacks.